### PR TITLE
Sign and Symptom Form: Change and Additional Information Needed

### LINKED ISSUE #705: Sign and Symptom Form: Change and Additional Information Needed
sign and symptom field

### PR DESCRIPTION
**Pull request by syedowaisali90**
Hi, @vincent-karuri I have created PR for review.
close #705

### CODE REVIEW

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json [hunk 1]
```diff
@@ -814,6 +877,28 @@
           "value": "true",
           "err": "{{patient-diagnostics-form-original.step2.has_fatigue.v_required.err}}"
         }
+      },
+      {
+        "key": "has_feel_malaise",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:882]: Could we rename the key to `feels_malaised` instead?

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json [hunk 2]
```diff
@@ -793,6 +834,28 @@
...
+      {
+        "key": "has_abdominal_pain",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "{{patient-diagnostics-form-original.step2.has_abdominal_pain.label}}",
+        "options": [
+          {
+            "key": "Yes",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:846]: Could we make the keys lowercase for uniformity?

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json [hunk 3]
```diff
@@ -551,6 +551,47 @@
...
+      {
+        "key": "has_fever_history",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "{{patient-diagnostics-form-original.step2.has_fever_history.label}}",
+        "options": [
+          {
+            "key": "Yes",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:582]: https://github.com/onaio/rdt-standard/pull/717/files#r559958891

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json [hunk 4]
```diff
@@ -814,6 +877,28 @@
...
+      {
+        "key": "has_feel_malaise",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "",
+        "openmrs_entity_id": "",
+        "type": "native_radio",
+        "label": "{{patient-diagnostics-form-original.step2.has_feel_malaise.label}}",
+        "options": [
+          {
+            "key": "Yes",
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json:890]: https://github.com/onaio/rdt-standard/pull/717/files#r559958891

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json (hunk 1 above)
Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json]: fixed.

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json (hunk 2 above)
Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json]: fixed.

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json (hunk 3 above)
Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json]: fixed.

File: rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json (hunk 4 above)
Comment (syedowaisali90) [reply rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form.json]: fixed.

**Review by vincent-karuri — APPROVED** [review]

**Pull request merged**